Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java

```diff
@@ -227,7 +227,8 @@ private ValidationResult checkTheIdentifier(Workpiece workpiece) throws DAOExcep
         User user = ServiceManager.getUserService().getAuthenticatedUser();
         String metadataLanguage = user != null ? user.getMetadataLanguage()
                 : Helper.getRequestParameter("Accept-Language");
-        return LanguageRange.parse(! metadataLanguage.isEmpty() ? metadataLanguage : "en");
+        return LanguageRange
+                .parse(Objects.nonNull(metadataLanguage) && !metadataLanguage.isEmpty() ? metadataLanguage : "en");
```

Comment:
I don't think that this change is necessary as `getMetadataLanguage()` is returning a String object in any case. See #3550 for changes in this method.